testgen/src/validator.rs

```diff
@@ -32,7 +36,7 @@ impl Validator {
     set_option!(proposer_priority, i64);
 
     /// Get a signer from this validator companion.
-    pub fn get_signer(&self) -> Result<Ed25519Signer, SimpleError> {
+    pub fn get_signer(&self) -> Result<Ed25519Keypair, SimpleError> {
```

Comment:
Perhaps this should be renamed to:

```suggestion
    pub fn get_keypair(&self) -> Result<Ed25519Keypair, SimpleError> {
```

Reply:
:+1: